src/main/java/com/capitalone/creditocr/model/ingest/LetterQueueProcessor.java

```diff
@@ -0,0 +1,206 @@
+package com.capitalone.creditocr.model.ingest;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LetterQueueProcessor {
+
+    // Holds the possible queues to be sorted in. The natural order indicates priority.
+    public enum queueType {FRAUD, VALIDATION, GENERAL, INQUIRIES, UNASSIGNED}
+    // Holds the number of matches for each queue
+    private Map<Enum, Integer> queueCount = new HashMap<Enum, Integer>(4);
+    // Holds the matching text and the queue that it belongs to
+    private Map<String, Enum> descriptions = new HashMap<String, Enum>(84);
+    // A list of each matching text that is found
+    private List<String> matchingDesctiptions = new ArrayList<String>();
+
+
+    /**
+     * This internal method sets up the counters and descriptions if not already.
+     */
+    private void initialize() {
```

Comment:
You might want to have this method empty the data structures each time you call it. Or, even better, add a new method to do this explicitly.